markdown/summary.Rmd

```diff
@@ -19,30 +46,29 @@ contact <- data.frame(contact1, contact2, contact3)
 
 # Percentage of End of Life Spent at Home or <br> 
 # in a Community Setting (Provisional)
-## Financial years ending 31 March 2011 to 2019
+## Financial years ending 31 March `r year(start_date) + 1` to `r year(end_date)`
 #### An Official Statistics publication for Scotland
 
-#### Publication date: 28 May 2019
-##### RESTRICTED STATISTICS: embargoed to 09:30 29/05/2019
+#### Publication date: `r format(pub_date, "%d %B %Y")`
+##### RESTRICTED STATISTICS: embargoed to 09:30 `r format(pub_date, "%d/%m/%Y")`
 
 ### About this release
 
-This release by the Information Services Division provides information on the percentage of time spent at home or in a community setting in the last six months of life. This is one of the measures used to support and determine progress towards the Scottish Government’s [“Strategic Framework for Action on Palliative and End of Life Care”](http://www.gov.scot/Resource/0049/00491388.pdf). Data are presented by financial year from 2010/11 to 2018/19, by NHS Board, Health and Social Care Partnership of residence and for Scotland. 
+This release by the Information Services Division provides information on the percentage of time spent at home or in a community setting in the last six months of life. This is one of the measures used to support and determine progress towards the Scottish Government’s [“Strategic Framework for Action on Palliative and End of Life Care”](http://www.gov.scot/Resource/0049/00491388.pdf). Data are presented by financial year from `r min(basefile$fy)` to `r max(basefile$fy)`, by NHS Board, Health and Social Care Partnership of residence and for Scotland. 
 
-Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
+Note that hospital data and National Records of Scotland deaths data are provisional for `r max(basefile$fy)`. This publication will be updated in October `r year(end_date)` with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
```

Comment:
I think because of this paragraph, it's hard to make the summary into 2 pages. We need to think about a way to do that.

Reply:
I've had a look at our previous publication summaries and the bottom margin is set to a really small number to make everything fit - is this ok to implement in our summary template?

Reply:
Another kind of related issue is with line spacing in the contact table - do you know if this will have to be altered manually (like the title) or can I do something in the template?

Reply:
I believe you can set the margin in the template. For the contact table, HSMR had the same issue. We couldn't find a way to customise the line spacing in the tables so have to manually adjust it.

Reply:
After knitting, I adjust the main title spacing as exactly 28 pt. Also adjusted "Financial years ending 31 March 2011 to 2019" to "Before 6 pt" and line spacing as "Single". Then things fit into 2 pages so we may not need to adjust the spacing for the contact table!